server/data/data-jpa/src/test/java/org/apache/james/jpa/healthcheck/JPAHealthCheckTest.java

```diff
@@ -0,0 +1,34 @@
+package org.apache.james.jpa.healthcheck;
+
+import org.apache.james.backends.jpa.JpaTestCluster;
+import org.apache.james.core.healthcheck.Result;
+import org.apache.james.mailrepository.jpa.JPAUrl;
+import org.apache.openjpa.persistence.EntityManagerFactoryImpl;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.persistence.EntityManagerFactory;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(JPAHealthCheckExtension.class)
+class JPAHealthCheckTest {
+
+    private JPAHealthCheck jpaHealthCheck;
+
+    @Test
+    void testWhenActive() {
+        final JpaTestCluster JPA_TEST_CLUSTER = JpaTestCluster.create(JPAUrl.class);
+        jpaHealthCheck = new JPAHealthCheck(JPA_TEST_CLUSTER.getEntityManagerFactory());
+        assertEquals(Result.healthy(jpaHealthCheck.componentName()),
```

Comment:
we use assertj in our code base, which is even recommanded by JUnit. https://junit.org/junit5/docs/current/user-guide/#writing-tests-assertions-third-party

Can you please also use it instead of junit assertions?